lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -43,10 +44,11 @@ def lint_lines(lines)
 
       def generate_errors(class_name, line_number)
         violated_rules(class_name).map do |violated_rule|
-          message = "Deprecated class `%s` detected matching the pattern `%s`. %s#{' ' + @addendum if @addendum}"
+          suggestion = " #{violated_rule[:suggestion]}".rstrip
+          message = "Deprecated class `%s` detected matching the pattern `%s`.%s #{@addendum}".strip
           {
             line: line_number,
-            message: format(message, class_name, violated_rule[:class_expr], violated_rule[:suggestion])
+            message: format(message, class_name, violated_rule[:class_expr], suggestion)
           }
         end
```

Comment:
This has the danger of removing extra spaces inside of the `class_expr`, `suggestion`, and `addendum` right? I don't know how comfortable I am with modifying their contents like that. It's possible that may intend to have 2 spaces in the provided string.